app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -294,8 +318,16 @@ fun MapScreen(
       setMultiTouchControls(true)
       // Limit the vertical scrollable area to avoid the user scrolling too far
       setScrollableAreaLimitLatitude(MapScreen.MAP_MAX_LATITUDE, MapScreen.MAP_MIN_LATITUDE, 0)
+
+      // Update the zoom level when the user zooms in or out
+      setOnTouchListener { _, _ ->
+        if (abs(zoomLevelDouble - zoomLevel) >= MapScreen.ZOOM_UPDATE_THRESHOLD)
+            zoomLevel = zoomLevelDouble
+        false
+      }
```

Comment:
Question: Is there not a listener that updates on the zoom change ?

Reply:
Sadly no...

Reply:
Okay, if it works, it works!